.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+-   repo: https://github.com/python/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.6
```

Comment:
What are the implications of the language_version option here? I still think we are targetting 3.5 as our minimum Py3 minor version.

Reply:
This is the version of python used to run black; changing it tells the pre-commit to use another version of python to run black.

Black will not change the python compatibility of the file.